lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
> The unicode header is only necessary for unicode characters in the actual source code, so it has been removed.

FWIW, while this change may not have been necessary for the original fix, I don't think there was any reason to remove it. We should tend to include it.

Reply:
From what I remember, that header is only necessary if the ACTUAL file has unicode characters in it, which I don't think they ever will?

Reply:
@kluo will have more info on this, I think.